docs/_doc/tutorials/wurst_for_vjass_users/page_1.md

```diff
@@ -0,0 +1,234 @@
+The wurst language server gives you inline code feedback, like a real IDE.
+
+It can feel unapproachable to be launching a map magicaly using a code editor, but it becomes second nature very fast.
+
+### Fragmented
+
+There's more than one JassHelper, and they're hardly different.
+Indeed, some very ubiquitous vJass libraries don't even work with the original JassHelper specification!
+
+### Typeless
+
+Because vjass can only expand your code into Jass, it can't make any guarantees about the type of values.
+This leads to tricky attaching and detaching of struct intances to timers and the like.
+
+Indeed, it's possible to cast instances of objects to integer in Wurst too - but that's frowned upon!
+
+Wurst at least gives you the capability to prevent entire classes of error like this.
+
+### No Scope for Improvement
+
+Maintaining JassHelper is *hard*, and the language specification designer isn't active anymore.
+
+Contrast that with the 33 commits so far this month on WurstScript - https://github.com/wurstscript/WurstScript/commits/master
+
+Of coure, there are costs to using language that's actively developed too - strange bugs are more numerous, and occasionally the compiler has a regression - but this is a small price to pay.
+
+## Why wurst?
+
+### integrated developer experience with *code* first
+
+As discussed above, wurst also provides a sleek development experience - but unlike vJass, it puts the code first.
+
+### A true compiler
+
+Wurst is a compiled language, which fundamentally gives it advantages and scope for enormous, leaping improvements, like:
+
+* updates that improve performance, by way of optimiser changes
+* debug information for failure scenarios - including in-game stack traces
+* Automatic nulling of variables to prevent tedious leaks
+
+### Higher levels of abstraction
+
+The basic wurst language implemented is a higher level language that lets you express more while writing less.
+
+Have you ever had an experience with vJass where you slammed out a hundred lines that implemented a stack or linked list, and it all felt very natural and rewarding?
+
+You'll have a similar experience when you write just a few lines of wurst and get the same functionality - these experiences are mind bending and highly valuable in the context of getting work done.
+
+### Standard Libary
+
+The convention in vJass is to share versioned libraries online as code snippets, but not in wurst.
+Why is that?
+
+It's not because wurst isn't worth sharing, or there's noone sharing it - wurst just fundamentally approaches the problem at a tooling level (much like cargo, pip, npm, etc).
+
+* Most of the basic requirements like data structures and familiar systems like damage detection are implemented in the standard library.
+`import DamageType` and done.
+Don't like the implementation?
+Implement your own and publish it for use with gradle, offer your own improvements upstream to the standard library, write your own standard library - why not?
```

Comment:
gradle doesn't really play a role with wurst though?
Maybe replace this with "the wurst dependency management" which the setup app will be once feature complete.